webapp/css/netpyne.less

```diff
@@ -593,4 +593,78 @@ div.flexlayout__layout {
 @keyframes spin {
   0% { transform: rotate(0deg); }
   100% { transform: rotate(360deg); }
-}
\ No newline at end of file
+}
+.editExperimentContainer {
```

Comment:
Why use `withStyles` and `netpyne.less` at the same time to style `EditExperiment.js`?